index.html

```diff
@@ -0,0 +1,163 @@
+				</select>
+			</div>
+			<div class="logo">
+				<a href="#">
+					<img src="images/logo.png" alt="logo"> 
+				</a>
+			</div>
+			<div class="cart-wrapper">
+				<a href="#">Cart (<span class="cart-quantity">3</span>)</a>
+			</div>
+		</div>
+	</div>
+	<div class="bottom-header">
+		<div class="wrapper">
+			<ul class="navigation">
+				<li><a href="#">Home</a></li>
+				<li><a href="#">Men</a>
+					<ul class="sub-navigation">
+						<li><a href="#">Jackets</a></li>
+						<li><a href="#">Shoes</a></li>
+						<li><a href="#">Coats</a></li>
+						<li><a href="#">Hats</a></li>
+					</ul>
+				</li>
+				<li><a href="#">woMen</a>
+					<ul class="sub-navigation">
+						<li><a href="#">Jackets</a></li>
+						<li><a href="#">Shoes</a></li>
+						<li><a href="#">Coats</a></li>
+						<li><a href="#">Hats</a></li>
+					</ul>
+				</li>
+				<li><a href="#">Lockbook</a></li>
+				<li><a href="#">about</a></li>
+				<li><a href="#">blog</a></li>
+			</ul>
+			<ul class="utility-menu">
+				<li><a href="#">wishlist</a></li>
+				<li><a href="#">search</a></li>
+			</ul>
+		</div>
+	</div>
+	<div class="advertisement-block">
+		<div class="wrapper">
+			<p>start selling your products or buy them from anywhere!</p>
+			<a href="#" class="close-button">close</a>
+		</div>
+	</div>
+</div>
+<div class="content">
+	<div class="wrapper">
+		<div class="product-image-container">
+			<div class="main-image-wrapper">
+				<img src="images/main-product-image.jpg" alt="main product image">
+			</div>
+			<ul class="thumbnails">
+				<li>
+					<a href="#" class="selected"><img src="images/product-thumbnail-1.jpg" alt="product thumbnail 1"></a>
+				</li>
+				<li>
+					<a href="#"><img src="images/product-thumbnail-2.jpg" alt="product thumbnail 2"></a>
+				</li>
+				<li>
+					<a href="#"><img src="images/product-thumbnail-3.jpg" alt="product thumbnail 3"></a>
+				</li>
+				<li>
+					<a href="#"><img src="images/product-thumbnail-4.jpg" alt="product thumbnail 4"></a>
+				</li>
+			</ul>
+		</div>
+		<div class="product-details-container">
+			<p class="product-text">Back to <a href="#" class="for-text-links">Woman<a/></p>
+			<h2 class="name-product">The Atelier Tailored Coat</h2>
+			<div class="rating">
+				<span class="rating-stars">☆</span><span class="rating-stars">☆</span><span class="rating-stars">☆</span><span class="rating-stars">☆</span><span class="rating-stars">☆</span>
+				<p class="product-rating-revies">3 Review(s) <span class="add-class-review">add a review</span></p>
+			</div>
+			<p class="product-price">&#36;499.00</p>
+			<ul class="product-secondary-infostore">
+				<li class="product-secondary-infostore">Availability :<span class="product-secondary-infostore-status">In stock</li>
+				<li class="product-secondary-infostore">Product code :<span class="product-secondary-infostore-status"> &#35;4657</li>
+				<li class="product-secondary-infostore">Tags:
+					<a href="#" class="for-text-links">Fashion</a>
```

Comment:
for-text-links - bad classname...it is a tag, please create classname based on this word